src/hooks/screen/useMessageHook.ts

```diff
@@ -1,21 +1,84 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import {Alert, NativeSyntheticEvent} from 'react-native';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The use of `any` type is generally discouraged in TypeScript as it defeats the purpose of static typing. If you're unsure about the type, consider using `unknown` instead which is safer. However, if possible, try to define a specific type or interface that describes the structure of your data.



<!-- This is an auto-generated comment by OSS CodeRabbit -->